assets/src/application/ui/calendars/dateDisplay/CalendarPageDate/style.css

```diff
@@ -0,0 +1,100 @@
+.ee-calendar-page-date-wrapper {
```

Comment:
I'd use css to scss convertor for content of this file: https://css2sass.herokuapp.com/

Reply:
ok will try